src/database/migrations/20240110072148-seed-data-to-notification-template.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+let addedRecordIds = [11, 12, 13, 14] // Keep track of added record IDs
```

Comment:
@VISHNUDAS-tunerlabs you can delete records based on the code

Reply:
@rakeshSgr kept like this because of above mentioned reason